examples/misc/svg_filter_line.py

```diff
@@ -52,7 +52,7 @@
 ax.set_ylim(0., 1.)
 
 # save the figure as a string in the svg format.
-from io import StringIO
+from six import StringIO
```

Comment:
er, no this is in the examples, we should not be teaching anyone to use our internal six.